Simplenote/UIActivityViewController+Simplenote.swift

```diff
@@ -16,7 +16,8 @@ extension UIActivityViewController {
 
         let print = SPSimpleTextPrintFormatter(text: content)
         let source = SimplenoteActivityItemSource(note: note)
+        let link = PDFLinkActivityItemProvider(content: content, filename: String(format: "%@.pdf", note.exportFilename()))
```

Comment:
Perhaps we can send over just the `filename` parameter, and append `+ ".pdf"` in the PDF ItemProvider? (since it'll always be PDF)